sys/auto_init/auto_init.c

```diff
@@ -24,6 +24,10 @@
 #include "bmp180.h"
 #endif
 
+#ifdef MODULE_BME280
```

Comment:
I know you took inspiration on the bmp180 implementation which has a flaw with auto_init (I apologize for this, feel free to blame me ;) ) : if you include the bmp280.h here when `MODULE_BME280` is loaded, you can end up with the device initialize twice.
An `auto_init_bme280` pseudo-module should be introduced to fix fix this problem. This is also a required change for the bmp180 (which should deserve another PR).

Also you forgot to call to `bme_280_auto_init()` in the `auto_init()` function.